contracts/setup_blending.sh

```diff
@@ -41,16 +32,46 @@ Sets up the environment for testing Blender application. Precisely:
 
 Make sure to have "cargo contract" installed (version 1.5).
 EOF
-  exit 0
 }
 
+while getopts r:n: flag
+do
+  case "${flag}" in
+    r) RUN_CHAIN=${OPTARG};;
+    n) NODE=${OPTARG};;
+    *)
+      usage
+      exit
+      ;;
+  esac
+done
+
+# defaults
+
+if [ -z ${RUN_CHAIN+x} ]; then
+  RUN_CHAIN=true
+fi
+
+if [ -z ${NODE+x} ]; then
+  NODE=ws://127.0.0.1:9944
+fi
```

Comment:
If you want to check if `RUN_CHAIN` was set and, if not. set to `true` then there's a shorthand for that:
```sh
RUN_CHAIN="${RUN_CHAIN:-true}"
NODE="${NODE:-ws://127.0.0.1:9944}"
```

Reply:
ok